docs/configurations.rst

```diff
@@ -28,18 +30,30 @@ Cheyenne
 
    jobqueue:
      pbs:
-       cores: 36
-       memory: 108GB
-       processes: 4
+       name: dask-worker
+       cores: 36                   # Total number of cores per job
+       memory: '109 GB'            # Total amount of memory per job
+       processes: 9                # Number of Python processes per job
+       interface: ib0              # Network interface to use like eth0 or ib0
+
+       queue: premium
```

Comment:
Is this definitely the right queue for everyone?  I'm inclined to leave these things out if they're not universally useful.

Reply:
Sounds good to me to have an as much as possible real example, so all except the project detail.